front_end/src/reducers/bookingReducer.js

```diff
@@ -50,8 +51,8 @@ export default function(state = initialState, action) {
       };
     case CLEAR_BOOKING:
       return { ...state, bookingConfirmationData: {} };
+    case CHANGE_RESERVATION:
```

Comment:
Please leave a TODO comment here to make it easier for the future programmer to know where to add code for future stories; assuming it is needed for future stories.

Reply:
It is not needed for future stories. That case is in there just to fall through and return the state.